fetch/main.js

```diff
@@ -1,17 +1,20 @@
 async function getData() {
-  try {
     const response = await fetch('https://jsonplaceholder.typicode.com/users');
+if (!response.ok){
+  new Error(`The reponse was unsucessful with the status Code - ${reponse.status}`)
+}
+  try {
     const data = await response.json();
     console.log(data);
   } catch (e) {
-    console.error(e);
+    console.error(Error);
   }
 }
 
 async function pokeData() {
   const response = await fetch('https://pokeapi.co/api/v2/pokemon/1')
-  if (response.ok === 'false'){
-    new Error('the reponse has failed error 404');
+  if (!response.ok){
```

Comment:
Yes, this is correct, but you need the `try` around the `await fetch`, too. Was it like this before and I missed it? Sorry!